### PR TITLE
fix accessibility issues on response query page

### PR DESCRIPTION
**Pull request by BlessingDeLaHaye-Oglafa**

### CODE REVIEW

File: app/uk/gov/hmrc/traderservices/views/EnterResponseTextView.scala.html [hunk 1]
```diff
@@ -31,14 +31,27 @@
...
+       
+        <div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="1000">
         @forms.formWithCSRF(action = postAction) {
-            @forms.inputCharacterCount(
-                form("responseText"),
-                hint = Some(messages("view.write-response-text.hint")),
-                label ="",
-                maxLength = Some(1000)
+            @forms.textarea(
+                form = form,
```
Comment (chandaksweta) [inline app/uk/gov/hmrc/traderservices/views/EnterResponseTextView.scala.html:38]: Does this not break the welsh char count work that I did couple of sprints back?

Comment (BlessingDeLaHaye-Oglafa) [reply app/uk/gov/hmrc/traderservices/views/EnterResponseTextView.scala.html:38]: Thanks Sweta, I'm having a look at it now :)

**Comment by BlessingDeLaHaye-Oglafa** [comment]
issues already fixed

**Comment by BlessingDeLaHaye-Oglafa** [comment]
issue already fixed

**Pull request closed without merging**